app/src/main/java/campus/tech/kakao/contacts/AddContact.kt

```diff
@@ -0,0 +1,86 @@
+package campus.tech.kakao.contacts
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.EditText
+import android.widget.TextView
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import androidx.constraintlayout.widget.ConstraintLayout
+
+class AddContact : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_add_contact)
+
+        val moreBtn = findViewById<TextView>(R.id.more)
+        val moreForm = findViewById<ConstraintLayout>(R.id.more_form)
+        val cancel = findViewById<TextView>(R.id.cancel)
+        val save = findViewById<TextView>(R.id.save)
+
+        moreBtn.setOnClickListener {
+            swapVisibility(moreBtn, moreForm)
+        }
+        cancel.setOnClickListener {
+            clickCancelEvent()
+        }
+        save.setOnClickListener {
+            val isCorrectInput = RequiredInputVerify()
+            if (isCorrectInput) {
+                clickSaveEvent()
+            }
+        }
+    }
+
+    fun swapVisibility(viewA: View, viewB: View) {
```

Comment:
보통은 함수를 분리하거나, 혹은 의미에 맞는 boolean을 인자로 전달하여 그에 따라 각 뷰의 visibility를 관리합니다.
```suggestion
    fun showMoreForm(isShow : Boolean) {
        moreForm.vibility = if(isShow) View.VISIBILE else View.GONE
        moreBtn.vibility = if(!isShow) View.VISIBILE else View.GONE
```